### DISCUSSION TITLE
Dashboard to identify busy volumes in an aggregate NABOX

### DISCUSSION THREAD

**Discussion by swordfish291** [NetApp/harvest#1137] [Q&A]
Hello Team,

I am using latest version of NaBox and I was looking for a dashboard that can provide IOPS/Throughput/Latency details of all the volumes in an aggregate. I am looking to identify busy volumes in an aggregate causing high disk utilization.

Thank you
Abdul

**Comment by rahulguptajss** [comment]
@swordfish291 Exisiting harvest dashboards do not provide this information. This should be easy to build with existing harvest metrics. Harvest volume performance metrics has aggregate information which can be used to get the relevant information. Let us know if you face any issues.

Example metrics:

`volume_labels{aggr="aggr0_sti115_vsim_ucs630n", cluster="sti115nscluster-1", datacenter="ZapiPerf", instance="localhost:12991", isEncrypted="false", isHardwareEncrypted="false", is_sis_volume="false", job="prometheus", node="sti115-vsim-ucs630n", protectedBy="not_applicable", protectionRole="unprotected", state="online", style="flexvol", svm="sti115-vsim-ucs630n", type="rw", volume="vol0"}`

**Reply by swordfish291** [reply]
Thanks for the response. This is way too technical for me. Is it possible to have this as a new enhancement in the next release?

**Reply by cgrinds** [reply]
hi @swordfish291, we'll take a look
